cli/api/ioutil2/ioutil.go

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
perhaps we should include a mention here of where this file came from?

https://github.com/youtube/vitess/blob/master/go/ioutil2/ioutil.go

Reply:
done